src/main.rs

```diff
@@ -37,94 +38,111 @@ Examples:
 boltzmann my-project --githubci=off --honeycomb --jwt"
 )]
-#[structopt(global_setting(ColoredHelp), global_setting(ColorAuto))]
 pub struct Flags {
-    #[structopt(long, help = "Enable redis")]
+    #[clap(long)]
+    /// Enable redis middleware
     redis: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable postgres")]
+    #[clap(long)]
+    /// Enable postgres middleware
     postgres: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable tracing via Honeycomb")]
+    #[clap(long)]
+    /// Enable tracing via Honeycomb
     honeycomb: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable GitHub actions CI")]
+    #[clap(long)]
+    /// Enable GitHub actions CI
     githubci: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable Nunjucks templates")]
+    #[clap(long)]
+    /// Enable Nunjucks templates
     templates: Option<Option<Flipper>>,
 
-    #[structopt(
-        long,
-        help = "Scaffold a project implemented in TypeScript",
-    )]
+    #[clap(long)]
+    /// Scaffold a project implemented in TypeScript
     typescript: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable csrf protection middleware")]
+    #[clap(long)]
+    /// Enable csrf protection middleware
     csrf: Option<Option<Flipper>>,
 
-    #[structopt(
-        long,
-        help = "Enable /monitor/status healthcheck endpoint; on by default"
-    )]
+    #[clap(long)]
+    /// Enable /monitor/status healthcheck endpoint; on by default
     status: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable /monitor/ping liveness endpoint; on by default")]
+    #[clap(long)]
+    /// Enable /monitor/ping liveness endpoint; on by default
     ping: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable jwt middleware")]
+    #[clap(long)]
+    /// Enable jwt middleware
     jwt: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable live reload in development")]
+    #[clap(long)]
+    /// Enable live reload in development
     livereload: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable OAuth")]
+    #[clap(long)]
+    /// Enable OAuth
     oauth: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable static file serving in development")]
+    #[clap(long)]
+    /// Enable static file serving in development
     staticfiles: Option<Option<Flipper>>,
 
-    #[structopt(long, help = "Enable asset bundling via ESBuild")]
+    #[clap(long)]
+    /// Enable asset bundling via ESBuild
     esbuild: Option<Option<Flipper>>,
 
     // Convenient option groups next. These aren't saved individually.
-    #[structopt(
+    #[clap(
         long,
-        help = "Enable website feature set (templates, csrf, staticfiles, jwt, livereload, ping, status)"
+        help = ""
     )]
+    /// Enable all features relevant to building websites
+    ///
+    /// This option group enables the templates, csrf, staticfile, jwt, livereload, ping, and
+    /// status options.
```

Comment:
🤔 I'm chewing on the "docstrings as help text" behavioral change from clap; docstrings and help text seem to have different audiences. It's nice to see that they can be separated, though!